html/manifest/display_override.json

```
@@ -0,0 +1,55 @@
+{
+  "html": {
+    "manifest": {
+      "display_override": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/display_override",
+          "spec_url": "https://wicg.github.io/manifest-incubations/#display_override-member",
+          "support": {
+            "chrome": {
+              "version_added": "89"
+            },
+            "chrome_android": {
+              "version_added": false
```

```suggestion
              "version_added": "89"
```

This rolled out for Android as well according to [crbug/1092667](https://bugs.chromium.org/p/chromium/issues/detail?id=1092667). While window controls overlay and tabbed application mode don't make sense on Android, you can still use it with the traditional values like `minimal-ui` if you want to customize your fallback chain.